scripts/create-package.sh

```diff
@@ -0,0 +1,268 @@
+    fi
+    if [ -n "$git_branch" ]; then 
+        sed -i -e "s|%GIT_BRANCH%|$git_branch|" "$readme_tmpl"
+    fi
+    if [ -n "$git_repo" ]; then
+        sed -i -e "s|%GITHUB_REPO%|$git_repo|"  "$readme_tmpl"
+    fi
+}
+
+install_translation()
+{
+    lng_dir=${pkg_dir}/translations
+    if [ "$platform" = "macos" ]; then
+        lng_dir=${macos_dst_dir}/Resources/translations
+    fi
+    # Prepare translation files
+    #
+    # Note:
+    #   We conciously drop the dialect postfix because no dialects are available.
+    #   (US was the default DOS dialect and therefore is the default for 'en').
+    #   Dialect translations will be added if/when they're available.
+    #
+    find contrib/translations -name '*.lng' |
+        while IFS= read -r src; do
+            target=$(basename "$src" | tr '[:upper:]' '[:lower:]')
+            install_file "$src" "$lng_dir/${target#??_}"
+        done
+}
+
+pkg_linux()
+{
+    # Print shared object dependencies
+    ldd "${build_dir}/dosbox"
+    install -DT "${build_dir}/dosbox" "${pkg_dir}/dosbox"
+
+    install -DT contrib/linux/dosbox-staging.desktop "${pkg_dir}/desktop/dosbox-staging.desktop"
+    DESTDIR="$(realpath "$pkg_dir")" make -C contrib/icons/ install datadir=
+}
+
+pkg_macos()
+{
+    # Note, this script assumes macos builds have x86_64 and ARM64 subdirectories
+
+    # Print shared object dependencies
+    otool -L "${build_dir}/dosbox-arm64/dosbox"
+    python3 scripts/verify-macos-dylibs.py "${build_dir}/dosbox-arm64/dosbox"
+
+    # Create universal binary from both architectures
+    mkdir dosbox-universal
+    lipo dosbox-x86_64/dosbox dosbox-arm64/dosbox -create -output dosbox-universal/dosbox
+
+    # Generate icon
+    make -C contrib/icons/ dosbox-staging.icns
+
+    install -d   "${macos_dst_dir}/MacOS/"
+    install      dosbox-universal/dosbox           "${macos_dst_dir}/MacOS/"
+    install_file contrib/macos/Info.plist.template "${macos_dst_dir}/Info.plist"
+    install_file contrib/macos/PkgInfo             "${macos_dst_dir}/PkgInfo"
+    install_file contrib/icons/dosbox-staging.icns "${macos_dst_dir}/Resources/"
+
+    sed -i -e "s|%VERSION%|${dbox_version}|"       "${macos_dst_dir}/Info.plist"
+}
+
+pkg_msys2()
+{
+    install -DT "${build_dir}/dosbox.exe" "${pkg_dir}/dosbox.exe"
+
+    # Discover and copy required dll files
+    ntldd -R "${pkg_dir}/dosbox.exe" \
+        | sed -e 's/^[[:blank:]]*//g' \
+        | cut -d ' ' -f 3 \
+        | grep -E -i '(mingw|clang)(32|64)' \
+        | sed -e 's|\\|/|g' \
+        | xargs cp --target-directory="${pkg_dir}/"
+}
+
+pkg_msvc()
+{
+    # Get the release dir name from $build_dir
+    release_dir=$(basename -- "$(dirname -- "${build_dir}")")
+
+    # Copy binary
+    cp "${build_dir}/dosbox.exe"  "${pkg_dir}/dosbox.exe"
+
+    # Copy dll files
+    cp "${build_dir}"/*.dll                  "${pkg_dir}/"
+    cp "src/libs/zmbv/${release_dir}"/*.dll  "${pkg_dir}/"
```

Comment:
This is failing for some reason with `cp: cannot stat 'src/libs/zmbv/x64/*.dll': No such file or directory`, any ideas?